src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java

```diff
@@ -65,20 +65,70 @@ public void testDefault() throws Exception {
 
     @Test
     public void testCast() throws Exception {
+        final DefaultConfiguration configurationForTestCast =
```

Comment:
please name it configurationTypeCast, please do the same approach for other naming.